public/index.html

```diff
@@ -40,18 +40,11 @@
             ga('send', 'pageview');
             });
         </script>
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-        <link rel="stylesheet" href="https://unpkg.com/higlass@1.11.3/dist/hglib.css">
         <title>Gosling.js Editor</title>
     </head>
 
     <body>
         <noscript>You need to enable JavaScript to run this app.</noscript>
         <div id="root"></div>
-        <script crossorigin type="text/javascript" src="https://unpkg.com/react@16/umd/react.development.js"></script>
-        <script crossorigin type="text/javascript" src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
-        <script crossorigin type="text/javascript" src="https://unpkg.com/pixi.js@5/dist/pixi.js"></script>
-        <script crossorigin type="text/javascript" src="https://unpkg.com/react-bootstrap@0.32.1/dist/react-bootstrap.js"></script>
-        <script crossorigin type="text/javascript" src="https://unpkg.com/higlass@1.11.7/dist/hglib.min.js"></script>
```

Comment:
None of these modules were externalized in the `webpack.config.editor.js`, so the amount of javascript downloaded when navigating to `gosling.js.org` is ~2x currently (since the modules are bundled in the output produced by webpack as well as imported separately here and not used)

Reply:
Oh, this was a huge mistake..